atlite/pv/irradiation.py

```diff
@@ -157,6 +157,7 @@ def TiltedIrradiation(
     trigon_model,
     clearsky_model,
     altitude_threshold=1.0,
+    irradiation="total",
```

Comment:
This might cause confusion, as `total` could also refer to `direct + diffuse + ground reflected + lost`. E.g. in the case of incident irradiation `total` is `direct+diffuse`.

Two suggestion:
1. Call it `combined` rather than `total`.
2. Add a docstring to `TiltedIrradiation` based on your docstring for the `irradiation(...)` method
```
    irradiation : str
        The irradiation quantity to be returned. Defaults to "total" for total
        irradiation. Other options include "direct" for direct irradiation,
        "diffuse" for diffuse irradation, and "ground" for ground irradiation.
```
where you explain what `total` (or `combined`) refers to.

Reply:
I think I'll stick with `total` for now, as it seems to me the resulting `xarray.Dataset` is currently renamed to `total tilted` anyway. I think renaming to `combined` would result in undue confusion. I tried adding a docstring though, but I must admit I'm not 100% clear on whether I got the explanations for all the input parameters correct.